tests/test_sca.py

```diff
@@ -407,7 +407,7 @@ def test_slow_software_composition_analysis(
 
     with open(syft_results, "r") as mock_scan_results:
         mock_syft.return_value = mock_scan_results.read()
-    cpu_software_composition_analysis(build_id)
+    cpu_software_composition_analysis(str(sb.uuid))
```

Comment:
Maybe it's better to consistently use sb.pk in all locations:
```suggestion
    cpu_software_composition_analysis(str(sb.pk))
```

Reply:
OK.